logstash-core/src/main/java/org/logstash/instrument/witness/ConfigWitness.java

```diff
@@ -92,6 +100,52 @@ public Snitch snitch() {
         return this.snitch;
     }
 
+    @Override
+    public void genJson(JsonGenerator gen, SerializerProvider provider) throws IOException {
+        new Serializer().innerSerialize(this, gen, provider);
```

Comment:
@jakelandis here too, we're calling this method a ton, we should make this `Serializer` something `static` since it has no state?